tools/test_torchserver.py

```diff
@@ -0,0 +1,57 @@
+import base64
+from argparse import ArgumentParser
+from io import BytesIO
+
+import matplotlib.pyplot as plt
+import mmcv
+import requests
+
+from mmseg.apis import inference_segmentor, init_segmentor
+
+
+def parse_args():
```

Comment:
Maybe you could add some description for the added test tool in docs:
https://github.com/open-mmlab/mmsegmentation/blob/master/docs/useful_tools.md#4-test-deployment